### PR TITLE
Add video module

### PR DESCRIPTION
**Pull request by JulianGro**
This PR adds a video module (thanks to Madders for that)
Currently this saves 137MiB as oppose to using gifs, while also allowing the use of audio and longer animations.

Specifically it changes:
- update readme for new dependency and instructions
- convert all gif animated files to webm (vp9) and mp4 (h264)
  - change all animated gif occurences in documentation to the new video module
  - remove now unused animated gif files
- convert some .md files to .rst to be able to use the newvideo directive
- update material-entity.rst
  - add updated example material to files
- add marker with "High Fidelity" keyword for later revisiting (need a script to replace the old "open in marketplace" example)

No warnings are generated that mention pages edited in this PR.
A build of this PR (currently 756d53d) can be viewed here: http://94.130.177.235/sphinx/%2372/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,4 +85,19 @@ Replace `xX` with your [language code](https://www.sphinx-doc.org/en/master/usag
...
 ## Using RST
 
-Most of our docs use RST. reStructuredText (RST) is the default plaintext markup language used by Sphinx. It is an extensible markup language, that is fully customizable. However, we don't (yet) have need of this, and will stick with the default set of directives for Vircadia's documentation. To learn more, refer to Sphinx's [reStructuredText Primer](https://www.sphinx-doc.org/en/2.0/usage/restructuredtext/basics.html).
+Most of our docs use RST. reStructuredText (RST) is the default plaintext markup language used by Sphinx. It is an extensible markup language, that is fully customizable. To learn more, refer to Sphinx's [reStructuredText Primer](https://www.sphinx-doc.org/en/2.0/usage/restructuredtext/basics.html).
+RST should be used for any real documentation, as Markdown only supports very basic directives.
+A valuable resource for RST is the [official documentation](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html).
+
+## Using videos
+
+When adding videos to the documentation, it is important to use h264 *and* vp9 to ensure they can be played it all major internet browsers.
```
Comment (ctrlaltdavid) [inline README.md:94]:
```suggestion
When adding videos to the documentation, it is important to use h264 *and* vp9 to ensure that they can be played it all major web browsers.
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]
Videos seem to need CSS styling to add whitespace under them. For example, the videos in the Create > Entities > Add a Material Entity page have no whitespace at their bottom before their following text starts. It's particularly noticeable that the heading "Use the materialData 
field" has no whitespace above it.

Additionally, videos should be centred horizontally, the same as images. For example, the videos on the Create > Entities > Change How Entities Look page are left-aligned whereas images are centred.

File: README.md [hunk 2]
```diff
@@ -80,4 +85,19 @@ Replace `xX` with your [language code](https://www.sphinx-doc.org/en/master/usag
...
+Most of our docs use RST. reStructuredText (RST) is the default plaintext markup language used by Sphinx. It is an extensible markup language, that is fully customizable. To learn more, refer to Sphinx's [reStructuredText Primer](https://www.sphinx-doc.org/en/2.0/usage/restructuredtext/basics.html).
+RST should be used for any real documentation, as Markdown only supports very basic directives.
+A valuable resource for RST is the [official documentation](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html).
+
+## Using videos
+
+When adding videos to the documentation, it is important to use h264 *and* vp9 to ensure that they can be played it all major web browsers.
+To convert videos and animated images to vp9 you can use following command in ffmpeg:
+    ```
```
Comment (ctrlaltdavid) [inline README.md:96]:
```suggestion

    ```
```
Blank line needed in order for command text to be displayed in its own container.

File: README.md [hunk 3]
```diff
@@ -80,4 +85,19 @@ Replace `xX` with your [language code](https://www.sphinx-doc.org/en/master/usag
...
+    ffmpeg -i INPUTFILE -c:v libvpx-vp9 -b:v 0 -crf 5 -vf "scale=-2:'min(600,ih)'" -cpu-used 5 -row-mt 1 -c:a libopus -b:a 96K _static/videos/OUTPUTFILE.webm
+    ```
+Or to convert to h264:
+    ```
```
Comment (ctrlaltdavid) [inline README.md:100]:
```suggestion

    ```
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

**Review by ctrlaltdavid — APPROVED** [review]

**Comment by JulianGro** [comment]
Closes #74

**Comment by JulianGro** [comment]
@kasenvr apparently I borked the history by rebasing on master. 18cccaa is the latest commit and has already been approved by @ctrlaltdavid

**Review by ctrlaltdavid — APPROVED** [review]

File: docs/source/conf.py [hunk 4]
```diff
@@ -191,3 +191,6 @@ def setup(app):
 
 # -- Markdown Setup ---------------------------------------------------------
 extensions = ['recommonmark']
+
+# -- Video Support ----------------------------------------------------------
+extensions = ['sphinxcontrib.video']
```
Comment (digisomni) [inline docs/source/conf.py:196]:
```suggestion

# -- Additional Extensions ---------------------------------------------------
extensions = ['recommonmark', 'sphinxcontrib.video']

```

Comment (digisomni) [reply docs/source/conf.py:196]: This will fix markdown parsing.

**Comment by digisomni** [comment]
After the merge conflict is solved and suggestion is applied, this can be merged with follow-up review on PR #105 to apply anything left.

**Review by digisomni — APPROVED** [review]
All good, so the only follow-up is PR 105. No rush on that though since the MD parsing was fixed in this PR.

**Pull request merged**